src/servers/mcp-server.ts

```diff
@@ -120,7 +126,17 @@ export class MCPServer extends BaseMCPServer {
                         readOnlyHint: true,
                         destructiveHint: false,
                     },
-                }
+                },
+                ...(this.isVersionGreaterThan1013() ? [{
```

Comment:
put comment on why the version check is needed

Reply:
Maybe instead of the version check, lets move to using the flag for this feature perhaps.